### DISCUSSION TITLE
failed to create server

### DISCUSSION THREAD

**Discussion by LCLabQU** [l3atbc-datadog/pushkin#99] [General] [answered]
Hi, 

So I got to 
$pushkin start;

and it did not work. I can send you the entire log but it seems the most important info is at the very end (below). Any suggestions what to try next? I tried running http://localhost and this takes me to Docker tutorial

Thanks!

Creating pushkin_server_1              ... error

ERROR: for pushkin_server_1  Cannot start service server: driver failed programming external connectivity on endpoint pushkin_server_1 (dff7c99f9a0aa34ded5fda3fa23203ce97194d24be771c06c8f745769c24b460): Bind for 0.0.0.0:80 failed: port is already allocated

ERROR: for server  Cannot start service server: driver failed programming external connectivity on endpoint pushkin_server_1 (dff7c99f9a0aa34ded5fda3fa23203ce97194d24be771c06c8f745769c24b460): Bind for 0.0.0.0:80 failed: port is already allocated
Encountered errors while bringing up the project.
something went wrong: undefined

**Comment by jkhartshorne** [comment, marked as answer]
See here: https://github.com/pushkin-consortium/pushkin/discussions/97

**Reply by jkhartshorne** [reply]
Please let me know if that solution doesn't work for you.

**Reply by LCLabQU** [reply]
Yes!  the kill command forced Docker to quit so I had to restart it. But I got my webpage :)
